apps/librelingo_json_export/librelingo_json_export/dictionary.py

```diff
@@ -60,4 +61,5 @@ def _define_words_in_sentence(course, sentence, reverse):
     """
     Converts a sentence into a list of definition objects.
     """
-    return [_define_word(course, word, reverse) for word in sentence.split()]
+    words = [p for p in re.split("( |\\{.*?\\}|'.*?')", sentence) if p.strip(' {}')]
```

Comment:
I would really extract this to a new function, something like `parse_phrase` that could probably be part of librelingo-utils, so that it can be shared between multiple modules, not just json-export